mlrun/model_monitoring/db/stores/v3io_kv/kv_store.py

```diff
@@ -703,3 +702,57 @@ def get_v3io_monitoring_apps_container(project_name: str) -> str:
     @staticmethod
     def _get_monitoring_schedules_container(project_name: str) -> str:
         return f"users/pipelines/{project_name}/monitoring-schedules/functions"
+
+    @staticmethod
+    def _extract_metrics_from_items(
+        items: list[dict[str, str]],
+    ) -> list[mlrun.common.schemas.model_monitoring.ModelEndpointMonitoringMetric]:
+        metrics: list[
+            mlrun.common.schemas.model_monitoring.ModelEndpointMonitoringMetric
+        ] = []
+        for app_item in items:
```

Comment:
I guess pick one name or the other?

Reply:
Do you mean just `app` or `item`?

Reply:
I mean `app_item in app_items` or `item in items`.

Reply:
76aa677